StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionSearchTableView.swift

```diff
@@ -112,7 +132,7 @@ final class InstitutionSearchTableView: UIView {
             loadingView.trailingAnchor.constraint(equalTo: loadingContainerView.trailingAnchor),
         ])
 
-        showTableFooterView(true)
+        showTableFooterView(false)
```

Comment:
Why this change?

Reply:
@vardges-stripe 

good question as this also tripped my 'is this a bug' detector; 

This is in the initializer. Previously, the footer would nearly always show (except some cases where we stop showing it like errors I believe). Now, the footer only gets shown in specific cases.